.vscode/tasks.json

```diff
@@ -287,6 +287,14 @@
       // NOTE: We don't have a way on Windows atm due to command with argument inside Run() not working
       // so we need to show logs outside of vscode (see #2454)
     },
+    {
```

Comment:
Can you run this within vscode GUI? I don't see it on mine (I know I won't be to test it on Windows though). Or maybe you need to define a launch config for that?

Reply:
Ya I choose run task from the command pallette and then it shows up.

Reply:
Ah ok ya would be better to expose it as a launch config. I can try to do it for you to test if you prefer too?

Reply:
Looks like launch.json is specifically for debugging, so you want to be able to debug while the tests are running ya? I guess it's just a matter of attaching to node and executed busted with the debug environment variable set.

Reply:
Right, yes, launch.json is for debugging. And yes that's what we want. Something similar to the `Neovim: Run` task I guess?

How does `busted` execute `nvim`? Is it only one execution of `nvim` for all tests (that are part of all the _spec.lua files) or does it spawn one `nvim` for each test? I guess we can only attach to it if only one execution of `nvim`? cc @pokey

Reply:
My understanding is that one instance of lua (so nvim) is run for each `*_spec.lua` file. So we could choose to just put everything in one, as busted does provide some nice mechanisms for organizing tests. Still might get a bit out of hand eventually though..

Reply:
https://github.com/cursorless-dev/cursorless/issues/2570